readme.md

```diff
@@ -52,9 +52,17 @@ Number of digits to appear after the seconds decimal point.
 ##### compact
 
 Type: `boolean`
```

Comment:
@sindresorhus This is how it renders right now : -

![image](https://cloud.githubusercontent.com/assets/3668034/8270765/5bf7f296-1810-11e5-9e14-479e393f83ae.png)

What do you mean by `needs to spaces at the end` ?